services/frontend-service/src/ui/components/LockDisplay/LockDisplay.scss

```diff
@@ -20,7 +20,7 @@ Copyright freiheit.com*/
     margin: $locks-list-row-spacing;
     justify-content: center;
     align-items: center;
-    height: $locks-list-row-height;
+    height: fit-content;
 }
 
 .lock-display-table {
```

Comment:
please give the commit message ~twice as much horizontal space as the other columns.